Lesson23/scripts.js

```diff
@@ -0,0 +1,19 @@
+function toggleAccordion(event) {
+    const header = event.target;
+    const content = header.nextElementSibling;
+
+    // Collapse all other accordion contents
+    const allContents = document.querySelectorAll('.accordion-content');
+    allContents.forEach(item => {
+        if (item !== content) {
+            item.style.maxHeight = null;
+        }
+    });
+
+    // Toggle the clicked accordion content
+    if (content.style.maxHeight) {
+        content.style.maxHeight = null;
+    } else {
+        content.style.maxHeight = '100px';
```

Comment:
Setting max height to 100px would work in this articular solution however should be avoided in real life. Instead, you'd need to use something that takes into account height of the content. E.g. `content.scrollHeight + 'px';`